network.tf

```
@@ -53,6 +53,7 @@ resource "google_compute_router" "vault-router" {
 
   bgp {
     asn = 64514
+    bgp = 20
```

Should this be `keepalive_interval`?

```suggestion
    keepalive_interval = 20
```